src/_includes/header/style.css

```
@@ -29,6 +29,7 @@
 
 .title {
   margin: 4.6em 0;
+  text-decoration: none;
```

```suggestion
  text-decoration: none;
  display: block;
```

…otherwise it breaks the mobile layout.